docs/content-services/interfaces/dialog.interface.md

```diff
@@ -34,18 +36,23 @@ interface DialogData {
 | ---- | ---- | ------------- | ----------- |
 | title | `string` |    | It will be placed in the dialog title section. |
 | headerIcon | `string` |    | It will be placed in header section. Should be used with Alert dialogs. (optional) |
-| description | `string` |    | It will be placed first in the dialog content section. (optional) |
+| description | `string` |    | It will be placed first in the dialog content section. Non-scrollable content. (optional) |
 | confirmButtonTitle | `string` | `COMMON.APPLY` | Confirmation action. After this, the dialog is closed and the `isConfirmButtonDisabled$` is set to `true`. (optional) |
 | cancelButtonTitle | `string` | `COMMON.CANCEL` | Cancellation action. After this, the dialog is closed |
 | isCancelButtonHidden | `boolean` | `false` | Toggles cancel button visibility. (optional) |
 | isCloseButtonHidden | `boolean` | `false` | Toggles close button visibility. (optional) |
 | isConfirmButtonDisabled$ | `Subject<boolean>` | `false` | Toggles confirm button disability. (optional) |
 | dialogSize | `DialogSize` | `Medium` | Set dialog size. Can be `Large`, `Medium`, `Alert`. (optional) |
+| contentText | `string` |    | Inserts a content text. (optional) |
+| contentComponent | `Type<any>` |    | Inserts a content component. (optional) |
 | contentTemplate | `TemplateRef<any>` |    | Inserts a content template. (optional) |
 | actionsTemplate | `TemplateRef<any>` |    | Inserts a template styled on the left. Should be used for additional `mat-button` style buttons. (optional) |
 | descriptionTemplate | `TemplateRef<any>` |    | Inserts a description template. (optional) |
+| additionalActionButton1 | `AdditionalDialogActionButton` |    | Inserts an additional button into the action bar on the left. (optional) |
```

Comment:
Maybe it will be better to just have additional action buttons as array? I doubt someone will use more than 2 of them and the code will look better

Reply:
done